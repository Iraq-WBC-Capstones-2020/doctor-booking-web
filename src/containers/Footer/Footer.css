footer {
  background-color: #014D87;
  min-height: 250px;
}

footer .footer-container {
  min-height: 250px;
}

footer .footer-container .row-container {
  min-height: 250px;
}

@media only screen and (max-width: 767px) {
  footer .first-col {
    margin-top: 1rem;
  }
}

footer a {
  color: white;
}

footer a:hover {
  color: white;
}
/*# sourceMappingURL=Footer.css.map */