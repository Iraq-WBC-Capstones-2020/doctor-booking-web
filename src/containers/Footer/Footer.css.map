{
    "version": 3,
    "mappings": "AAEA,AAAA,MAAM,CAAC;EACL,gBAAgB,ECFJ,OAAO;EDGnB,UAAU,EAAE,KAAK;CAmBlB;;AArBD,AAGE,MAHI,CAGJ,iBAAiB,CAAC;EAChB,UAAU,EAAE,KAAK;CAIlB;;AARH,AAKI,MALE,CAGJ,iBAAiB,CAEf,cAAc,CAAC;EACb,UAAU,EAAE,KAAK;CAClB;;AAGH,MAAM,MAAM,MAAM,MAAM,SAAS,EAAE,KAAK;EAV1C,AAWI,MAXE,CAWF,UAAU,CAAC;IACT,UAAU,EAAE,IAAI;GACjB;;;AAbL,AAeE,MAfI,CAeJ,CAAC,CAAC;EACA,KAAK,EAAE,KAAK;CAIb;;AApBH,AAiBI,MAjBE,CAeJ,CAAC,AAEE,MAAM,CAAC;EACN,KAAK,EAAE,KAAK;CACb",
    "sources": [
        "Footer.scss",
        "../../styles/_variables.scss"
    ],
    "names": [],
    "file": "Footer.css"
}