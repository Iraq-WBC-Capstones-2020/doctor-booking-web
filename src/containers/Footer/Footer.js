import React from 'react';
import { Container, Row, Col } from 'react-bootstrap';
function Footer() {
  return (
    <footer className="text-white">
      <Container className="h-100 footer-container">
        <Row className="h-100 align-content-center row-container">
          <Col className="mb-3 first-col" md="3">
            <h5>Doctor Booking</h5>
            <p>A website to book doctor appointment and save you the time</p>
          </Col>
          <Col md="3" className="mb-3 offset-md-2">
            <h5>Links</h5>

            <ul className="list-unstyled">
              <li>
                <a href="#!">Home</a>
              </li>
              <li>
                <a href="#!">About</a>
              </li>
              <li>
                <a href="#!">FAQ</a>
              </li>
            </ul>
          </Col>

          <Col md="3" className="mb-3 offset-md-1">
            <h5>Services</h5>

            <ul className="list-unstyled">
              <li>
                <a href="#!">Book an appointment</a>
              </li>
              <li>
                <a href="#!">Clinic registration</a>
              </li>
            </ul>
          </Col>
        </Row>
      </Container>
    </footer>
  );
}

export default Footer;
