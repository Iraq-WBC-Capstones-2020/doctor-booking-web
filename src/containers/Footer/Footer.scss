@import '../../styles/variables';

footer {
  background-color: $darkerBlue;
  min-height: 250px;
  .container {
    min-height: 250px;
    .row {
      min-height: 250px;
    }
  }

  @media only screen and(max-width: 767px) {
    .first-col {
      margin-top: 1rem;
    }
  }
  a {
    color: white;
    &:hover {
      color: white;
    }
  }
}
