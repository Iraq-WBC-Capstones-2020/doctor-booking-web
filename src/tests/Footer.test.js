import React from 'react';
import { render } from '@testing-library/react';
import Footer from '../containers/Footer/Footer';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
test('footer is rendering ', () => {
  render(
    <MemoryRouter>
      <Footer />
    </MemoryRouter>
  );
});

test('footer renders correctly', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <Footer />
      </MemoryRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
