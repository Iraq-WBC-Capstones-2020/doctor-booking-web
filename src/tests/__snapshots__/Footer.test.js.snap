// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer renders correctly 1`] = `
<footer
  className="text-white"
>
  <div
    className="h-100 container"
  >
    <div
      className="h-100 align-content-center  row"
    >
      <div
        className="mb-3 first-col col-md-3"
      >
        <h5>
          Doctor Booking
        </h5>
        <p>
          A website to book doctor appointment and save you the time
        </p>
      </div>
      <div
        className="mb-3 offset-md-2 col-md-3"
      >
        <h5>
          Links
        </h5>
        <ul
          className="list-unstyled"
        >
          <li>
            <a
              href="#!"
            >
              Home
            </a>
          </li>
          <li>
            <a
              href="#!"
            >
              About
            </a>
          </li>
          <li>
            <a
              href="#!"
            >
              FAQ
            </a>
          </li>
        </ul>
      </div>
      <div
        className="mb-3 offset-md-1 col-md-3"
      >
        <h5>
          Services
        </h5>
        <ul
          className="list-unstyled"
        >
          <li>
            <a
              href="#!"
            >
              Book an appointment
            </a>
          </li>
          <li>
            <a
              href="#!"
            >
              Clinic registration
            </a>
          </li>
        </ul>
      </div>
    </div>
  </div>
</footer>
`;
